WebApplicationTests/UnitTest1.cs

```diff
@@ -0,0 +1,379 @@
+using Autofac.Extras.Moq;
+using health_clinic_class_diagram.Backend.Model.Survey;
+using Moq;
+using System;
+using System.Collections.Generic;
+using WebApplication.Backend.Repositorys;
+using WebApplication.Backend.Services;
+using WebApplication.Backend.Util;
+using Xunit;
+
+namespace WebApplicationTests
+{
+    public class UnitTest1
+    {
+            [Fact]
+            public void AddNewSurveyTest()
+            {
+                var stubRepository = new Mock<ISurveyRepository>();
+                var result = true;
+                Survey surveyTest = new Survey
+                ("001", "Bozo Bozic", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3", "3");
```

Comment:
You can move test objects for stubs to a field or into a constants class to reuse in all tests

Reply:
Separate assign, act and assert parts with an empty line 😄